pkg/reconcile/pipeline/handler/collect/impl.go

```diff
@@ -63,11 +64,22 @@ func BindingDefinitions(ctx pipeline.Context) {
 		util.MergeMaps(anns, service.Resource().GetAnnotations())
 
 		for k, v := range anns {
+			if !strings.HasPrefix(k, binding.AnnotationPrefix) || k == ProvisionedServiceAnnotationKey {
+				continue
+			}
+
 			definition, err := makeBindingDefinition(k, v, ctx)
 			if err != nil {
-				continue
+				condition := apis.Conditions().NotCollectionReady().
```

Comment:
you can use here `notCollectionReadyCond` function available within the same package, for improved reading.

Reply:
For some reason that I really don't understand, using `notCollectionReadyCond` seems to cause this to break.  For reference, see the CI logs for commits 678672f and 44c2abf.